spdx-convertor/convert.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2024 Joas Schilling <coding@schilljs.com>
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+if ($argc < 2 || in_array('--help', $argv) || in_array('-h', $argv)) {
+	die("convert.php [--dry-run] <path>\n");
+}
+
+$isDryRun = $argv[1] === '--dry-run';
+$path = $isDryRun ? $argv[2] : $argv[1];
```

Comment:
Lazy me called the script without a trailing slash (`php convert.php ../../nextcloud/server/apps-extra/user_usage_report`)

and ended up with a `user_usage_reportAUTHORS.md` in my apps-extra folder :roll_eyes:

Reply:
Fixed